src/inverseRobot.js

```diff
@@ -8,6 +8,17 @@
 
 function inverseRobot(robot) {
   // write code here
+  const rightRobot = {};
```

Comment:
Consider using a more descriptive name for 'rightRobot', such as 'invertedRobot'.